Cargo.toml

```diff
@@ -27,6 +27,7 @@ default = ["use_std"]
 use_std = []
 align = []
 rustc-dep-of-std = ['align', 'rustc-std-workspace-core']
+extra_traits = []
```

Comment:
We might want to start by using more fine grained features here, and having an `extra_traits` feature that enables them all.